src/evidently/ui/app.py

```diff
@@ -42,9 +43,17 @@ def run_local(
     secret = secret or os.environ.get(EVIDENTLY_SECRET_ENV)
     if secret is not None:
         config.security = TokenSecurityComponent(token=SecretStr(secret))
+
+    if return_litestar_app:
```

Comment:
i think it's better to do the other way around. eg create `configure_app` with the same signature as `run_local` that does everything except `run(config)` and make `run_local(...): run(configure_app(...))` and `litestar_app(): create_app(configure_app())`

Reply:
fixed